### PR TITLE
Add missing j9vm CRIU NLS samples

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

**Comment by pshipton** [comment]
@singh264 fyi

File: runtime/nls/j9vm/j9vm.nls [hunk 1]
```diff
@@ -2316,6 +2316,7 @@ J9NLS_VM_CRIU_CRAC_INCOMPATIBLE_SETTING.user_response=Remove one of the options
 
 J9NLS_VM_CRIU_J9_GET_PROCESS_START_TIME_FAILURE=j9sysinfo_get_process_start_time failed with errno=%zi
 # START NON-TRANSLATABLE
+J9NLS_VM_CRIU_J9_GET_PROCESS_START_TIME_FAILURE.sample_input_1=-34
```
Comment (keithc-ca) [inline runtime/nls/j9vm/j9vm.nls:2319]: I wondered about that when reviewing https://github.com/eclipse-openj9/openj9/pull/18680, but I saw other (integer) template parameters that didn't have samples, so I assumed it was unnecessary for integer formats. See entries at lines 2200, 2207, 2214 to list just the first few. If a sample is needed, it should be added for all.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: runtime/nls/j9vm/j9vm.nls (hunk 1 above)
Comment (pshipton) [reply runtime/nls/j9vm/j9vm.nls:2334]: Pushed another commit to resolve the other missing samples.

**Comment by keithc-ca** [comment]
Could you squash this, please?

**Comment by pshipton** [comment]
Squashed

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**